### DISCUSSION TITLE
2023/06/book-not-down/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#83] [General]
# 2023/06/book-not-down/

https://xiangyun.rbind.io/2023/06/book-not-down/

**Comment by XiangyunHuang** [comment]
希望两年后再看这些，看到的都是幼稚。记录这些心路历程的目的是见证成长，同时也给自己一个鞭策。

**Reply by XiangyunHuang** [reply]
2025-09-20 两年后，我只记得拔了一个 flag ，就是做很笨很费时不太情愿做的事，尝试了下收集整理中国各市级行政区的年末常住人口，结果详见[中国常住人口分布图（2023 年市级）](https://xiangyun.rbind.io/2025/09/china-population/)

**Comment by yourlulu1997** [comment]
真希望早日看到空间数据科学及其应用这本书

**Reply by XiangyunHuang** [reply]
目前《空间数据科学及其应用》是本笔记，距离书还比较遥远。不过，《R 语言数据分析实战》的两章[预测核辐射强度的分布](https://bookdown.org/xiangyun/data-analysis-in-action/analyze-spatial-data.html) 和 [高斯过程回归](https://bookdown.org/xiangyun/data-analysis-in-action/gaussian-processes-regression.html)分别从频率派和贝叶斯派角度对空间数据建模。此外，博客页面中有不少空间数据操作、分析、可视化的文章。
